awx/ui_next/src/screens/Job/JobDetail/JobDetail.test.jsx

```diff
@@ -29,8 +29,8 @@ describe('<JobDetail />', () => {
     }
 
     assertDetail('Status', 'Successful');
-    assertDetail('Started', mockJobData.started);
-    assertDetail('Finished', mockJobData.finished);
+    assertDetail('Started', '8/8/2019, 7:24:18 PM');
```

Comment:
hardcoded navigator lang needed here too

Reply:
@jlmitch5 nodes always going to use en-US. unless you compile it with special flags. The tests are passing in CI.

 I've updated the utility tests to use a hard-coded lang just to keep it explicit but I don't really see the need to do it here...

Reply:
got it, makes  sense